### DISCUSSION TITLE
Total pellet usage and total kWh for Energy dashboard

### DISCUSSION THREAD

**Discussion by realthargor** [dominikamann/oekofen-pellematic-compact#33] [Q&A] [answered]
First of all thanks for the awesome integration.
Works flawless. :)

One thing I'm struggeling with is to add the pellet usage to the engery dashboard.
I know it is called "Gas usage" but technically it just expects a metric in kWh which can be calculated from the pellet usage.

I tried to achieve this by setting up a utility helper and a template metric to calculate the kWh from the kg.
And then there is this mess, which might help:
https://community.home-assistant.io/t/calculate-cumulative-total-sensor-from-daily-total-sensor/335275/6

I was wondering if you know how to achieve this or if it even might be something the integration itself could provide directly.

**Comment by dominikamann** [comment]
Hi @realthargor,
i also tought about adding it as "Gas" to the Energy-Dashboard. This would be very usefull.
As soon as I have time I will check your link and play arround ... .  I will keep you up to date.

**Comment by dominikamann** [comment, marked as answer]
@realthargor see https://github.com/dominikamann/oekofen-pellematic-compact/discussions/36
